tests/test_mongod_binary.py

```diff
@@ -0,0 +1,137 @@
+"""Test pytest_motor.mongod_binary."""
+import tempfile
+from pathlib import Path
+
+from pytest import MonkeyPatch, mark, raises
+
+from pytest_motor.mongod_binary import MongodBinary
+
+# Archives imitate direcotry structure of real ones
+# but do NOT contain real mongod binaries
+# P.S. Linux and MacOS structure is the same
+MONGODB_MACOS_TAR = Path(__file__).parent.parent / "test_data" / \
+    "mongodb-macos-x86_64-4.4.6.tgz"
+MONGODB_WINDOWS_ZIP = Path(__file__).parent.parent / "test_data" / \
+    "mongodb-windows-x86_64-4.4.6.zip"
+
+
+@mark.filterwarnings("ignore::Warning")
+def test_mongo_platform(monkeypatch: MonkeyPatch) -> None:
```

Comment:
It would probably be better to parameterize this test and other tests with `pytest.mark.parametrize`.